File: PythonCyber/CountWords.py

class CountWords:
    # Python program to count the number of occurrence
    # of a comp in the given string given string

    def countOccurences(str_, comp):
        #print(str_,comp,'<<<<')
        str_=str_.lower()
        comp=comp.lower()
        comp=comp.split()
        # split the string by spaces in a
        a = str_.split(" ")
        # search for pattern in a
        count = 0
        for w in comp:
            for i in range(0, len(a)):
            # if match found increase count
                if (w == a[i]):
                    count = count + 1
        return count

if __name__ == '__main__':
    str = "A computer science     portaL portal geeks  "
    comp = "computer science"
    print(Countcomps.countOccurences(str, comp))

File: WEB-INF/classes/CountWords.java
package CT;

public class  CountWords
{
	public static int main(String input, String strFind) 
	{

		input=input.toLowerCase();
		strFind=strFind.toLowerCase();
		
			String a[] = input.split("\\s+"); 

			// search for pattern in a 
			int count = 0; 
			for (int i = 0; i < a.length; i++) 
			{ 
			// if match found increase count 
			if (strFind.equals(a[i])){ 
				count++; 
			}
        System.out.println(a[i]+"	Total occurrences: " + count);
			} 


        
        System.out.println("Total occurrences: " + count);
		return count;		 
	} 


	public static void main(String[] args) 
	{
		System.out.println(main("sajid sajid sajid nashu","sajid"));
	}

}
